### PR TITLE
fix make incompatibility

### PR DESCRIPTION
**Pull request by linlynn**
the errors are:
Makefile:442: *** mixed implicit and normal rules: deprecated syntax
Makefile:1610: *** mixed implicit and normal rules: deprecated syntax

The problem is that we did stuff like this: config %config: ... The solution was simple - the above was split into two with identical prerequisites and commands.

### CODE REVIEW

**Comment by linlynn** [comment]
@tfangit ,please let me know if this pull request is good to you

File: meta-aspeed/recipes-kernel/linux/files/patch-2.6.28.9/0029_fix_make_incompatibility.patch [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ # Build targets only - this includes vmlinux, arch specific targets, clean
+@@ -1607,7 +1610,7 @@
+ 	$(Q)$(MAKE) $(build)=$(build-dir) $(target-dir)$(notdir $@)
+ 
+ # Modules
+-/ %/: prepare scripts FORCE
++/: prepare scripts FORCE
+ 	$(cmd_crmodverdir)
+ 	$(Q)$(MAKE) KBUILD_MODULES=$(if $(CONFIG_MODULES),1) \
+ 	$(build)=$(build-dir)
```
Comment (tfangit) [inline meta-aspeed/recipes-kernel/linux/files/patch-2.6.28.9/0029_fix_make_incompatibility.patch]: Shouldn't have two sections like 'config and %config' above?

**Comment by linlynn** [comment]
@tfangit thanks for the comments,not sure how I was missing that ,anyway,update patch ,please take a look

**Comment by linlynn** [comment]
do you have action to review all existing pull request?

**Comment by linlynn** [comment]
@theopolis  I resolved the conflicts and here is PR: https://github.com/facebook/openbmc/pull/27

**Pull request closed without merging**